code/controllers/subsystem/vote.dm

```diff
@@ -71,6 +71,12 @@ SUBSYSTEM_DEF(vote)
 					choices[GLOB.master_mode] += non_voters.len
 					if(choices[GLOB.master_mode] >= greatest_votes)
 						greatest_votes = choices[GLOB.master_mode]
+			else if(mode == "map")
+				for (var/client/c in non_voters)

```

Comment:
This is associative list

Reply:
huh, how did I even do that wrong. also should it rather be a did not vote instead of voting preferred map?